idea/src/META-INF/plugin.xml

```diff
@@ -2622,6 +2622,16 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.SelfAssignmentInspection"
+                     displayName="Self assignment"
+                     groupPath="Kotlin"
+                     groupName="Probable bugs"
+                     enabledByDefault="true"
+                     cleanupTool="true"
```

Comment:
Please do not set `cleanupTool = true` if you are not absolutely sure that your inspection should be included into "Code Cleanup" action.